app/src/main/java/com/asfoundation/wallet/analytics/RakamAnalyticsSetup.kt

```diff
@@ -1,12 +1,27 @@
 package com.asfoundation.wallet.analytics
 
 import io.rakam.api.Rakam
+import org.json.JSONException
+import org.json.JSONObject
 
 class RakamAnalyticsSetup : AnalyticsSetUp {
-
   private val rakamClient = Rakam.getInstance()
 
-  override fun setDefaultWallet(walletAddress: String) {
+  override fun setUserId(walletAddress: String) {
     rakamClient.setUserId(walletAddress)
   }
+
+  override fun setGamificationLevel(level: Int) {
+    var superProperties: JSONObject? = rakamClient.superProperties
```

Comment:
No need for type explicity here

Reply:
Also the conditions can be done like bellow:
`val superProperties = rakamClient.superProperties?: JSONObject()`